UnitWorld_shared/shared/game/play/MobileUnit.cpp

```diff
@@ -17,24 +17,35 @@ void MobileUnit::actualize()
 
     _speed += _acceleration;
     _speed.maxAt(maximumSpeed());
-    position() += _speed;
+    position(position() + _speed);
```

Comment:
There is now a code style inconsistency between:
- `_speed += _acceleration`
- `position(position() + _speed)`

I had to pause to realize they were quite similar operations.

Reply:
well I don't have access to the position private member of my parent, while I do have access to the speed. do you recommend using the position way of incrementing but for both? I usually have a preference for using my private members when possible

Reply:
I guess this is part of your coding style standards. No problem, then.